client/a8c-for-agencies/sections/woopayments/primary/woopayments-dashboard/style.scss

```diff
@@ -2,13 +2,64 @@
 @import "@wordpress/base-styles/mixins";
 @import "@wordpress/base-styles/variables";
 
+$woocommerce-purple-50: #720EEC;
```

Comment:
Nitpicking: I've noticed that this color appears in several areas already. Could we standardize it and update the other components to use it as global variable?